include/Server.hpp

```diff
@@ -37,6 +37,8 @@ class Server
     }
   };
 
+  static Server &getInstance();
```

Comment:
```suggestion
  static Server &getInstance() noexcept;
```
(need to update the implementation prototype in the .cpp too ofc.)